### ISSUE TITLE
OPS supported/accelerated with MSLK for fp8/fp4 modes

### ISSUE THREAD

**Issue by Alexey234432** [meta-pytorch/MSLK#155]
Hi,

Could someone advice me on which ops supported/accelerated by mslk for Fp8/fp4 compute?

In the source code I see that there is support for conv and GEMM (matrix multiply) - but what about other ops such as plus, minus, log, softmax? 
What I am trying to do is to re-write PyTorch model with fp8 compute - is it possible to go beyond conv/linear layers with help of MSLK?

Thanks

**Comment by cthi** [comment]
Hi - for FP4 we support gemm. For FP8 we support gemm and conv as you noticed.

For the other ops you mentioned - we don't have support for those. Some of those would likely be supported in PyTorch already, but you could also look into using a high level DSL like Helion to write ones yourself.
